CHANGELOG.md

```diff
@@ -1,3 +1,103 @@
+### citus v11.3.0 (April 13, 2023) ###
+
+PR #6845 : When creating a HTAB we need to use HASH_COMPARE  flag in order to
+set a user defined comparison function.
+* Fixes an uninitialized memory access in shard split API
+
+PR #6837 : fixes update propagation bug when `citus_set_coordinator_host` is
+called more than once
+* Fixes update propagation bug when `citus_set_coordinator_host` is called more
+  than once.
+
+PR #6771 : Adds control for background task executors involving a node
+* Adds control for background task executors involving a node
+
+PR #6820 : Add upgrade/downgrade paths between v11.2.2 and v11.3.1
+* PR description that will go into the change log, up to 78 characters
+
+PR #6794 : Move cleanup record test to upgrade schedule
+* Move cleanup record test to upgrade schedule
+
+PR #6725 : Multi tenant monitoring
+* Adds views that monitor statistics on tenant usages
+
+PR #6805 : Do not override combinefunc of custom aggregates with common names
+* Fix an issue that caused some queries with custom aggregates to fail
+
+PR #6728 : stabilize metadata syncing
+* Fixes memory leak and max allocation block errors during metadata syncing.
+
+* Introduces nontransactional mode for metadata sync.
+
+* Introduces the GUC `citus.metadata_sync_mode` to switch sync modes
+
+PR #6738 :  Identity column implementation refactorings
+* Prevents using identity columns on data types other than `bigint` on
+  distributed tables
+
+* Prevents using `alter_distributed_table` and `undistribute_table` UDFs when a
+  table has identity columns
+
+* Fixes a bug that prevents enforcing identity column restrictions on worker
+  nodes
+
+PR #6756 : Schedule parallel shard moves in background rebalancer by removing
+task dependencies between shard moves across colocation groups.
+* This PR removes the task dependencies between shard moves for which the shards
+  belong to different colocation groups. This change results in scheduling
+  multiple tasks in the RUNNABLE state. Therefore it is possible that the
+  background task monitor can run them concurrently.
+
+PR #6776 : Propagate CREATE/ALTER/DROP PUBLICATION statements
+* Propagate CREATE/ALTER/DROP PUBLICATION statements for distributed tables
+
+PR #6802 : Handle identity columns properly in the router planner
+* Fixes a bug with insert..select queries with identity columns
+
+PR #6775 : Add appropriate locks for MERGE to run in in parallel
+* PR description that will go into the change log, up to 78 characters
```

Comment:
does that fix a bug that we've introduced in this release or not?

Reply:
cc: @tejeswarm

Reply:
Done.